src/core/redux/Store.tsx

```diff
@@ -15,13 +20,18 @@ export const initStore = (initialState: {} = {}) => {
     /* eslint sort-keys: "error" */
     reducer: {
       articles: articlesReducer,
+      attendanceEvents: attendanceEventsReducer,
+      attendees: attendeesReducer,
       careerLocations: careerLocationsReducer,
       careerOpportunities: careerOpportunitiesReducer,
       companies: companiesReducer,
       events: eventsReducer,
       jobTypes: jobTypesReducer,
+      onlineGroups: onlineGroupsReducer,
       payments: paymentsReducer,
+      ruleBundles: ruleBundlesReducer,
       shop: shopReducer,
+      transactions: transactionsReducer,
     },
     /* eeslint sort-keys: "off" */
```

Comment:
```suggestion
    /* eslint sort-keys: "off" */
```
I made a typo, can we fix it here? 😅